### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by Madhvik007**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
...
+## How to Run?
+
+To run the application you need to clone the project and run the HTML file.
+
+## Screenshot
+
+![image](https://user-images.githubusercontent.com/72201530/193838132-4e22f252-ce18-42ef-8066-b1e392d784a8.png)
+
+
+## 🚀 About Me
```
Comment (ppfeiler) [inline README.md:30]: I am not sure if we really need a "About Me" section as this project evolves to an community project. Same for the "Links" section. 

@DhanushNehru what do you think?

Comment (DhanushNehru) [reply README.md:30]: Yeah that's what I told to remove the about section.

Missed to see this one, will remove it

Comment (DhanushNehru) [reply README.md:30]: Yeah I told to remove the about me section

Comment (ppfeiler) [reply README.md:30]: Honestly, I am not sure what else is new on this pull request? Just blank lines?

**Comment by selemondev** [comment]
The issue has finally been resolved! Huge thanks to everyone who has contributed to this pull request.

**Pull request closed without merging**